test/accessibility/run-pa11y.sh

```diff
@@ -56,6 +56,10 @@ caseload_url='caseload'
 urls+=("$host/team/$teamId/$caseload_url")
 urls+=("$host/ldu/$lduId/$caseload_url")
 
+# Reduction
+reduction_url='reduction'
```

Comment:
Look like it should be `reductions`. And another url for the add page

Reply:
Yep, will do that.